.github/workflows/tests.yml

```
@@ -22,7 +22,7 @@ jobs:
         - ["3.9",   "py39"]
         - ["3.10",  "py310"]
         - ["pypy-3.9", "pypy3"]
-        - ["3.10",  "docs"]
+#        - ["3.10",  "docs"]  # disable as readthedocs builds it
```

If we do this, then there should be a cross-reference in the `.readthedocs.yaml` file to update this file when its Python version is updated. Also RTD is configured to use 3.11.

```suggestion
        - ["3.10",  "docs"]
```

---

I disabled this because the docs generation does not create any coverage output. At the moment and with the issue https://github.com/coverallsapp/github-action/issues/216 that I opened, this makes the build run.

---

Oh, I misinterpreted what was going on. Nevermind. This is fine.